front/package.json

```diff
@@ -35,7 +35,8 @@
     "detect-deadcode": "ts-prune -p app/tsconfig.json -i '(node_modules|utils/testUtils/rtl.tsx|utils/permissions|i18n|/*/*/__mocks__)' | (! grep -v 'used in module')",
     "dependency-cruiser": "depcruise app  --validate",
     "storybook": "storybook dev -p 6006",
-    "storybook:build": "storybook build"
+    "storybook:build": "storybook build",
+    "postinstall": "node ./internals/scripts/copy-comp-lib-build.js"
```

Comment:
So this is the trick: after `npm install` completes, we copy over the `src` folder of the component library into the `internals/jest` folder in the `front` folder. We then point `jest` in the main repo to this file instead of to the component library link in our `package.json`.

The downside is that this means that you must have done `npm install` before running `npm run test`. If not, you will get an error message explaining that `internals/jest/cl2-component-library/src` was not found.

The alternative is to run the copy script every time before running `npm run test`. That would solve this issue but maybe be overkill... what do you think?

Reply:
Nice work! A couple of remarks:
- Is it possible to point to the component library source folder directly instead of copy-ing it, given it's in the same repo?
- If we must copy it, should we add it to git ignore to make sure we don't commit it?
- Maybe we can add the `npm install` to the test script to make sure it works every time you run the tests? Not sure how much slower that would be, though, perhaps not worth it?

Reply:
> Is it possible to point to the component library source folder directly instead of copy-ing it, given it's in the same repo?

Unfortunately not, it gives the same hook error because you have the same dependency conflict with the component library dependencies. If you copy the folder over, it uses the dependencies of the main repo instead.

> If we must copy it, should we add it to git ignore to make sure we don't commit it?

I already added it :)

> Maybe we can add the npm install to the test script to make sure it works every time you run the tests? Not sure how much slower that would be, though, perhaps not worth it?

Yeah I'll just add the copy script itself to `npm run test`. Then you also don't have to run `npm i` anymore. The `src` folder of the component library is pretty small anyway so I think copying it won't add more than a second to the test run time.

Reply:
> I think the other challenge will be that if someone is running and modifying a test that depends on the library, they have to run npm install each time. 

That also solves this issue

Reply:
Sounds good! Thanks for your work on this, Luuc :)